src/common/options.cc

```diff
@@ -2578,7 +2578,11 @@ std::vector<Option> get_global_options() {
 
     Option("osd_map_message_max", Option::TYPE_INT, Option::LEVEL_ADVANCED)
     .set_default(40)
-    .set_description(""),
+    .set_description("maximum number of OSDMaps to include in a single message"),
+
+    Option("osd_map_message_max_bytes", Option::TYPE_SIZE, Option::LEVEL_ADVANCED)
+    .set_default(10_M)
```

Comment:
would this need be tuned for a worst-case like a large cluster w/many PGs, all upmapped?

If we get to 10M per map we'd only be sending full maps

Reply:
a low limit just means there'd be one (incremental) map per message.

the only time we use full maps is when there is a discontinuity (client has an epoch older than our oldest map)